wheatley/bot.py

```diff
@@ -67,7 +68,14 @@ def __init__(self, tower: RingingRoomTower, row_generator: RowGenerator, do_up_d
 
         self._is_ringing = False
         self._is_ringing_rounds = True
-        self._should_start_method = False
+        # This is used as a counter - once `Go` or `Look To` is received, the number of rounds left
+        # is calculated and then decremented at the start of every subsequent row until it reaches
+        # 0, at which point the method starts.  We keep a counter rather than a simple flag so that
+        # calls can be called **before** going into changes when Wheatley is calling (useful for
+        # calling the first method name in spliced and early calls in Original, Erin, etc.).  The
+        # value `None` is used to represent the case where we don't know when we will be starting
+        # the method (and therefore there it makes no sense to decrement this counter).
```

Comment:
Possibly - it's far from ideal for me because Vim will line wrap correctly but still treat the whole line as one, which is extremely unintuitive.  Also, I'd like to keep the lines all under 100 or 110 chars so that I don't have to suppress Pylint's errors.